### PR TITLE
Allow Actions to run CI/CD 

### PR DESCRIPTION
**Pull request by zhuliice**
Turn on Actions to run CI/CD on this repository.

### CODE REVIEW

**Comment by zhuliice** [comment]
build by Actions is here: https://github.com/Autodesk-Forge/forge-api-dotnet-design.automation/pull/12/checks?check_run_id=1330077325, under `checks` above too

File: .github/workflows/dotnet-core.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 2.2.108
```
Comment (szilvaa-adsk) [inline .github/workflows/dotnet-core.yml:19]: Shouldn't this be 3.1? I thought 2.2 is already deprecated by MS.

Comment (zhuliice) [reply .github/workflows/dotnet-core.yml:19]: We are still at `netstandard2.0`
from https://docs.microsoft.com/en-us/dotnet/standard/net-standard, .net core 3.0 supports 2.1, so I think the lastest version `dotnet-version: 3.1.301` won't work

Do we want to upgrade to `netstandard2.1`?

Comment (szilvaa) [reply .github/workflows/dotnet-core.yml:19]: I see. No, let's stay on netstandard2.0

**Review by szilvaa — APPROVED** [review]

**Pull request merged**